### DISCUSSION TITLE
Welcome to goldendict-ng Discussions!

### DISCUSSION THREAD

**Discussion by xiaoyifang** [xiaoyifang/goldendict-ng#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 332891547** [comment]
请问ao-4.dll文件能发一份吗，网上找了好久都找不到

**Comment by xiaoyifang** [comment]
https://github.com/xiaoyifang/goldendict/blob/staged/winlibs/lib/msvc/ao-4.dll

**Comment by chenao1233** [comment]
求助：安装了20221030发布的新版本，win和MAC，词典内容显示部分在调成较大字体，在输入单词后，为什么内容显示会先显示较小字体，闪了一下再变成较大字体？原来安装的win旧版本（两三年前）不会，MAC的第一次用不知道旧版本会不会。我的系统是WIN10，MACOS是Monterey12.4。谢谢

**Reply by xiaoyifang** [reply]
Can you upload a screen record of the operation?

**Reply by chenao1233** [reply]
MACOS Monterey 12.4 ，Thank






https://user-images.githubusercontent.com/117447873/200115957-c4a8d3c0-2afd-4f11-a62b-4089c1709d8d.mp4

**Comment by xiaoyifang** [comment]
发个goldendict about的截图。里面有qt的版本，猜测你用的是qt5.15的版本，用 qt6.x 版本试一下

**Reply by chenao1233** [reply]
Update from qt5.15.2 to qt6.3.2, problem solved, thank.

**Comment by tue-thong** [comment]
Please help. I could not download "https://github.com/xiaoyifang/goldendict-ng/releases/download/v25.05.0-Release.2a2b0e16/GoldenDict-ng-25.05.0-Qt6.6.3-Windows-installer.exe or "https://github.com/xiaoyifang/goldendict-ng/releases/download/v25.05.0-Release.2a2b0e16/GoldenDict-ng-25.05.0-Qt6.7.2-Windows-installer.exe" from the page: "https://github.com/xiaoyifang/goldendict-ng/releases/tag/v25.05.0-Release.2a2b0e16". I already tried some ways as Copilot's instructions, but still did not work. Thank you so much. I am mr tue-thong, a business & management trainer in Ho Chi Minh city.
